spec/routing/work_packages_spec.rb

```diff
@@ -66,10 +64,10 @@
     end
   end
 
-  it 'should connect GET /work_packages/:id to work_packages#show' do
+  it 'should connect GET /work_packages/:id to work_packages#index' do
     expect(get('/work_packages/1')).to route_to(controller: 'work_packages',
-                                                action: 'show',
-                                                id: '1')
+                                                action: 'index',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.